pkg/storaged/storage.scss

```diff
@@ -500,6 +500,34 @@ td.storage-action {
     font-size: var(--pf-global--FontSize--xs);
 }
 
+.sidepanel-row:not(.pf-c-empty-state):hover {
+    cursor: pointer;
+}
+
+/* Add hover styling to ct-tables and sidepanels */
+
+.sidepanel-row:not(.pf-c-empty-state):hover,
+.storage-mounts .ct-table tbody tr:hover {
+    box-shadow: var(--pf-global--BoxShadow--sm);
+}
+
+.storage-mounts .ct-table {
+    // Reset color style
+    --ct-color-list-hover-text: all;
+}
+
+.sidepanel-row:hover .sidepanel-row-name,
+#others .sidepanel-row:hover .sidepanel-row-devname,
+.storage-mounts tr:hover td[data-label=Name],
+.storage-mounts tr:hover td[data-label=Server] {
+    // ct color variable is for the ct-table-based widgets
```

Comment:
These can go away if you use `<Button variant="link" component="link"` for the rows content.